native/storybook/ThemeStyles.js

```diff
@@ -0,0 +1,48 @@
+/*
```

Comment:
This should be named `themeAttributes` - Upper camel case is reserved for components